src/scenes/Projects/Update/ProjectWorkflowDialog.tsx

```diff
@@ -87,9 +87,13 @@ export const ProjectWorkflowDialog = ({
       <Grid container direction="column" spacing={1}>
         {transitions.map((transition, i) => (
           <Tooltip
-            title={`This will change the project step to ${displayProjectStep(
-              transition.to
-            )}`}
+            title={
+              !transition.enabled
+                ? 'The project step cannot be changed to Completed until all engagements have a terminal status'
```

Comment:
We should not assume this is the reason the transition is not enabled. The API should provide the reason why it's not enabled.

Reply:
What if this is the only one?

Reply:
If it feels unnecessary, trust me it's not. There's lots of validation we'd like to add in the future, that's dependent upon the current state. i.e. "the project cannot become active if it doesn't have start or end dates defined."

Reply:
What if we check `enabled` and `to`? Is your main concern keeping the FE clean?

Reply:
Though I guess this does seem like more of an API responsibility.